api/src/main/java/org/openmrs/notification/db/hibernate/HibernateAlertDAO.java

```diff
@@ -112,7 +112,7 @@ public List<Alert> getAlerts(User user, boolean includeRead, boolean includeExpi
 		List<Predicate> predicates = new ArrayList<>();
 
 		if (user != null && user.getUserId() != null) {
-			predicates.add(cb.equal(root.join("recipients").get("recipientId"), user.getUserId()));
```

Comment:
Do you mind explaining how this change is related to the ticket?

Reply:
This was a side effect of switching to the JPA. I reverted this and updated the PR.